src/stories/purchase-flow.stories.svelte

```diff
@@ -111,9 +111,7 @@
             </ModalHeader>
             <StatePresentOffer {...args} />
           </Aside>
-          <Main brandingAppearance={args.brandingInfo.appearance}>
-            <StateNeedsPaymentInfo {...args} />
-          </Main>
+          <StateNeedsPaymentInfo {...args} />
```

Comment:
I agree, I'm not happy with this solution. But I wanted to avoid code repetition here and in the `state-needs-payment-info.stories.svelte`. Is it ok if we rename the utils component to something like `StateNeedsPaymentInfoWithPurchaseResponse`?